src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -351,4 +355,16 @@ private boolean shouldBeProcessed(DetailNode curNode) {
         return javadocTokens.contains(curNode.getType());
     }
 
+    /**
+     * This method determines if a check should process javadoc containing unclosed html tags.
+     * This method must be overridden in checks extending {@code AbstractJavadocCheck} which
+     * are not supposed to process javadoc containing unclosed html tags.
+     * @return true if the check should or can process javadoc containing unclosed html tags;
+     *     false otherwise
+     * @see JavadocUtils#hasUnclosedTag(DetailNode)
+     */
+    public boolean doesCheckAcceptJavadocWithUnclosedHtmlTag() {
```

Comment:
Please make simple overridable methods like this near the top like https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractCheck.java#L89 . No one will find it on the bottom.